package.json

```diff
@@ -72,7 +71,7 @@
     "gatsby-transformer-remark": "^5.11.1",
     "gatsby-transformer-sharp": "^4.11.0",
     "html-entities": "^2.3.2",
-    "i18next": "^21.6.15",
+    "i18next": "^21.6.16",
```

Comment:
so I guess you smuggle in some dependency upgrade to save some time next Monday? 😄